usage-collection/src/main/proto/usage.proto

```diff
@@ -1588,8 +1618,8 @@ message CDPProxyDetails {
 
 // Show services for which the customer has provided their own DNS zones
 message CDPOwnDnsZones {
-  // Whether DNS is provided for the postgres by the customer or not
-  bool postgres = 1;
+    // Whether DNS is provided for the postgres by the customer or not
+    bool postgres = 1;
 }
```

Comment:
Not a big deal but think this extra indentation was not intentional. This file seem to use 2 spaces elsewhere. :smile:

Reply:
this is copy paste from thunderhead